make-all.sh

```diff
@@ -23,6 +23,6 @@ else
     printf "\n${RED}|*******| SOURCE COMPILATION FAILED |*******|${NC}\n\n"
     exit 1
 fi
-# cp src/TerminalFighter ../
+cp src/TerminalFighter/TerminalFighter ../
```

Comment:
So with your new changes to src/ directories. the executable used to build on...
build/src/<here>(TerminalFighter)
But now, it builds in
build/src/TerminalFighter/<here>(TerminalFighter)

On your old PR, we both missed it, but the ./make-all script was actually not copying over the executable. I realized this when I noticed my changes would affect the compile, but it would not affect the actual executable.